src/Application/Auth/Token.php

```diff
@@ -45,7 +51,7 @@ public static function create(string $personId): string
      * @param LoggerInterface   $logger
      * @return bool Whether the token is valid for the given person.
      */
-    public static function check(string $personId, string $jws, LoggerInterface $logger): bool
+    public static function check(string $personId, bool $complete, string $jws, LoggerInterface $logger): bool
```

Comment:
Would be nice to have some docs for the new param var here too
`@param bool $complete ...`